src/uniswapV3/UniswapV3Wrapper.sol

```diff
@@ -93,7 +93,7 @@ contract UniswapV3Wrapper is BaseWrapper, IUniswapV3FlashCallback {
         uint256 amount1 = asset == asset1 ? amount : 0;
 
         _activePool = pool;
-        pool.flash(address(this), amount0, amount1, data);
+        pool.flash(address(this), amount0, amount1, abi.encode(asset, amount, data));
```

Comment:
if we store the second asset and the fee we can kill `_activePool`, saving an sstore and a ssload

Reply:
*if we encode

Reply:
Done